README.md

```diff
@@ -27,7 +27,11 @@ store.init({}).then(function() {
 ```
 
 # DB Api
-The `$fh.db` backend utilizes the storage engine provided by the FeedHenry Cloud API, wrapped by the Simple Store API and is activated when the `WFM_USE_MEMORY_STORE` enviroment variable is set to `false`. The general configuration is done via environment variables.
+The `$fh.db` backend utilizes the storage engine provided by the FeedHenry Cloud API, wrapped by the Simple Store API .
```

Comment:
Maybe something like:

The persistent storage functionality is provided by the $fh.db Cloud API. This API is provided by the RHMAP Platform to Cloud Applications. (Include link)